install-steps.md

```diff
@@ -1,193 +1,821 @@
+to the `/etc/hosts` file
+
+~~~sh
+<OUTPUT_OMITTED>
+<NIC2-IP> openshift-worker-1.openshift.example.com openshift-worker-1
+<NIC2-IP> openshift-worker-2.openshift.example.com openshift-worker-2
+~~~
+
+NOTE: Remove the provisioner.openshift.example.com entry as it is replaced by worker-2
+
+## Create DHCP Reservations on DHCP Server (Option 1)
+
+1. Login into DHCP server via `ssh`
+2. Edit /etc/dhcp/dhcpd.hosts
+
+    ~~~
+    host openshift-worker-2 {
+         option host-name "worker-2";
+         hardware ethernet <NIC2-mac-address>;
+         option domain-search "openshift.example.com";
+         fixed-address <ip-address-of-NIC2>;
+      }
+    ~~~
+    NOTE: Remove the provisioner host entry as it is replaced by the openshift-worker-2
+3. Restart `dhcpd` service: 
+   ~~~sh
+   systemctl restart dhcpd
+   ~~~
+
+## Create DHCP Reservations using `dnsmasq` (Option 2) 
+
+Within the server hosting the `dnsmasq` service append the following DHCP
+reservation to the `/etc/dnsmasq.d/example.dns` file
+
+~~~sh
+<OUTPUT_OMITTED>
+dhcp-host=<NIC2-mac-address>,openshift-worker-1.openshift.example.com,<ip-of-worker-1>
+dhcp-host=<NIC2-mac-address>,openshift-worker-2.openshift.example.com,<ip-of-worker-2>
+~~~
+
+NOTE: Remove the provisioner host entry as it is replaced by the openshift-worker-2
+
+Once the change as been added and saved, restart the `dnsmasq` service
+~~~sh
+systemctl restart dnsmasq
+~~~
+
+## Deploy the Provisioner node as a Worker Node using Metal3
+
+In order to convert the provisioner node as a worker node ensure the following:
+
+1. `ssh` to a system that can access the out of band management network of the current provisioner node
+2. Poweroff the node using the `ipmitool` and confirm the provisioning node is powered off
+
+    ~~~sh
+    ssh <server-with-access-to-management-net>
+    # Use the user, password and Management net IP adddress to shutdown the system
+    ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> -p 6230 power off
+    # Confirm the server is powered down
+    ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> -p 6230 power status
+    Chassis Power is off
+    ~~~
+3. Get `base64` strings for the Out of band Management credentials. In the example user is `root`, password is `calvin`
+
+    ~~~sh
+    # Use echo -ne, otherwise you will get your secrets with \n which will cause issues
+    # Get root username in base64
+    echo -ne "root" | base64
+    # Get root password in base64
+    echo -ne "calvin" | base64
+    ~~~
+4. Configure the BaremetalHost `bmh.yaml`
+
+    ~~~yaml
+    ---
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: openshift-worker-2-bmc-secret
+    type: Opaque
+    data:
+      username: ca2vdAo=
+      password: MWAwTWdtdC0K
+    ---
+    apiVersion: metal3.io/v1alpha1
+    kind: BareMetalHost
+    metadata:
+      name: openshift-worker-2
+    spec:
+      online: true
+      bootMACAddress: <NIC1-mac-address>
```

Comment:
I've always needed to set `hardwareProfile: unknown` in the workers to make this work

Reply:
Just for reference: I've tried without setting up the hardwareProfile, and that will end up in hardwareProfile being set to unknown, that being said, configuring it to unknow by default can be a good idea.

Reply:
I have made similar observances.

Reply:
added `hardwareProfile: unknown` to the workers